psgo.go

```diff
@@ -80,7 +80,7 @@ func translateDescriptors(descriptors []string) ([]aixFormatDescriptor, error) {
 
 var (
 	// DefaultDescriptors is the `ps -ef` compatible default format.
-	DefaultDescriptors = []string{"user", "pid", "ppid", "pcpu", "etime", "tty", "time", "args"}
+	DefaultDescriptors = []string{"user", "pid", "ppid", "pcpu", "etime", "tty", "time", "comm"}
```

Comment:
Aren't you reverting the previous change?

Reply:
That wasn't intended at all. Thanks for catching that! I must have screwed up the rebase.